pcsx2/USB/Win32/Config_usb.cpp

```diff
@@ -64,6 +64,18 @@ void SelChangedAPI(HWND hW, int port)
 	changedAPIs[std::make_pair(port, devName)] = *it;
 }
 
+void SelChangedSubtype(HWND hW, int port)
+{
+	int sel = SendDlgItemMessage(hW, port ? IDC_COMBO_WHEEL_TYPE1_USB : IDC_COMBO_WHEEL_TYPE2_USB, CB_GETCURSEL, 0, 0);
```

Comment:
You can use const here, didn't check others but would be nice to change them too if there are any that can be changed.